Fakebook.Posts/Fakebook.Posts.UnitTests/PostTests.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Posts.Domain.Models;
+using Xunit;
+namespace Fakebook.Posts.UnitTests
+{
+
+    public class PostTests{
+        private const int Id = 1;
+        [Fact]
+        public void Post_Constructor_Pass(){
+            //arrange
+                const string Content = "This is some content";
+                const string userEmail = "damion.silver@test.com";
+            //act
+            Post newpost = new Post(userEmail,Content);
+
+            //assert
+                Assert.NotNull(newpost);
+        }
+    
+
+         [Fact]
+        public void Post_EmailIsNotNull(){
+            //arrange
+                const string Content = "This is some content";
+                const string userEmail = "damion.silver@test.com";
+            //act
+            Post newcomment = new Post(userEmail, Content);
+
+            //assert
+                Assert.NotNull(newcomment.UserEmail);
```

Comment:
weird indentation throughout this PR. stick to the preexisting style. it's easy, every editor has a command for it